### ISSUE TITLE
Request project be added to Runestone Academy

### ISSUE THREAD

**Issue by semelech** [RunestoneInteractive/rs#317]
This is a request for our Bridges project be added to Runestone Academy.
The repo URL: https://github.com/semelech/BRIDGES

**Comment by bnmnetp** [comment]
There are some problems that need to be fixed.

1. Your project.ptx file has the runestone target outside of the <targets>
2. The Runestone publication file should be the same as publication.ptx with the **addition** of the platform tag.

**Comment by semelech** [comment]
I fixed the project.ptx file and added the runestone target inside of the <targets> tag. I also fixed runestone.ptx to match publication.ptx and then added <platform host="runestone"/> inside the <html> tag. Let me know if there is anything else to fix. Thanks

**Comment by bnmnetp** [comment]
You are just missing your format and source tags in the runestone target.  I temporarily added them and was able to get it to build.

**Comment by semelech** [comment]
Do I need to permanently add these tags? Was it the format and source tags that I was missing? After that please let me know what the next steps would be to see the finished product. Thanks

**Comment by bnmnetp** [comment]
Yes,  The build process needs to know that for the target Runestone we are building html format and it needs to know where to find the source files.

**Comment by bnmnetp** [comment]
OK, your book is available at : https://runestone.academy/ns/books/published/bridgesds/frontmatter.html?mode=browsing

You can register for it using bridgesds

I have made you an author, so you can now use https://author.runestone.academy/author as described in the PreTeXt guide.

**Comment by bnmnetp** [comment]
Closing — this was resolved at the time. The Bridges book built and went live at `bridgesds`, and @semelech was set up as an author. The fixes needed were the `runestone` target inside `<targets>`, its `format` and `source` tags, and a `runestone.ptx` publication file matching `publication.ptx` plus the `<platform host="runestone"/>` tag.
